Makefile

```diff
@@ -10,6 +10,10 @@ ifeq ($(OUTPUT_DIR),)
 OUTPUT_DIR = bin
 endif
 
+ifeq ($(NXDK_ENV),)
```

Comment:
`bin/activate` is for interactive use and we don't use it in build scripts, please don't make it required.